tests/FakeItEasy.Specs/DummyCreationSpecs.cs

```diff
@@ -911,6 +911,41 @@ protected override IList<T> CreateCollectionOfDummy<T>(int count)
 
     public class NonGenericDummyCreationSpecs : DummyCreationSpecsBase
     {
+        [Scenario]
+        public void IsByRefLikeCreation(object? dummy, Exception? exception)
+        {
+            "Given a byref-like value type"
+                .See(nameof(ReadOnlySpan<char>));
+
+            "When a dummy of that type is requested"
+                .x(() => exception = Record.Exception(() => dummy = Sdk.Create.Dummy(typeof(ReadOnlySpan<char>))));
+
+#if CAN_CREATE_ISBYREFLIKE
+            "Then no exception is thrown"
+                .x(() => exception.Should().BeNull());
+
+            "And the dummy was created"
+                .x(() => dummy.Should().NotBeNull());
+#elif OBLIVIOUS_TO_ISBYREFLIKE_CREATION
+            "Then it throws an exception of type DummyCreationException"
+                .x(() => exception.Should().BeAnExceptionOfType<NotSupportedException>());

```

Comment:
The step description doesn't match the actual assertion. Is it `DummyCreationException` or `NotSupportedException`?

Reply:
blush